src/set_theory/game/domineering.lean

```diff
@@ -180,12 +180,12 @@ instance short_L : short domineering.L := by { dsimp [domineering.L], apply_inst
 -- run_cmd tactic.whnf `(by apply_instance : decidable (domineering.one ≤ 1)) >>= tactic.trace
 
 -- dec_trivial can handle most of the dictionary of small games described in [conway2001]
-example : domineering.one ≈ 1 := dec_trivial
-example : domineering.L + domineering.L ≈ 1 := dec_trivial
-example : domineering.L ≈ pgame.of_lists [0] [1] := dec_trivial
-example : (domineering ([(0,0), (0,1), (0,2), (0,3)].to_finset) ≈ 2) := dec_trivial
-example : (domineering ([(0,0), (0,1), (1,0), (1,1)].to_finset) ≈ pgame.of_lists [1] [-1]) :=
-  dec_trivial.
+-- example : domineering.one ≈ 1 := dec_trivial
```

Comment:
The decidability instances here won't reduce in the kernel anymore since they're defined by well-founded recursion.

cc @semorrison

Reply:
This is worth mentioning explicitly in the comments.